Templates/CDS-CSharp/run.csx

```diff
@@ -0,0 +1,68 @@
+﻿// Usage of this template is subject to license terms of the 
+// Microsoft Common Data Service Software Development Kit (https://go.microsoft.com/fwlink/?linkid=842862).
+
+#load ".\SecurityHelper.csx"
+#load ".\TraceWriterTelemetryBridge.csx"
+
+using Microsoft.CommonDataService;
+using Microsoft.CommonDataService.CommonEntitySets;
+using Microsoft.CommonDataService.Configuration;
+using Microsoft.CommonDataService.ServiceClient.Security;
+using Microsoft.CommonDataService.ServiceClient.Security.Credentials;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+class EnvironmentValues
+{
+    internal const string Tenant = "[[Replace with AAD tenant for CDS value]]";
+    internal const string EnvironmentId = "[[Replace with PowerApps environment ID value]]";
+    internal const string ApplicationId = "[[Replace with Function application ID value]]";
```

Comment:
Are we getting these values from EasyAuth later? If yes, are we tracking in another work item?